src/blueprints/tv2/factories/tv2-sisyfos-audio-timeline-object-factory.ts

```diff
@@ -0,0 +1,79 @@
+import { Tv2AudioTimelineObjectFactory } from '../value-objects/factories/tv2-audio-timeline-object-factory'
+import {
+  SisyfosChannelsTimelineObject,
+  SisyfosResynchronizeTimelineObject,
+  SisyfosType
+} from '../../timeline-state-resolver-types/sisyfos-types'
+import { Tv2SisyfosLayer } from '../value-objects/tv2-layers'
+import { DeviceType } from '../../../model/enums/device-type'
+import { Tv2BlueprintConfiguration } from '../value-objects/tv2-blueprint-configuration'
+import { EmptyTimelineObject } from '../../timeline-state-resolver-types/abstract-types'
+
+const enum SisyfosFaderState {
+  OFF = 0,
+  ON = 1,
+  VOICE_OVER = 2
+}
+
+export class Tv2SisyfosAudioTimelineObjectFactory implements Tv2AudioTimelineObjectFactory {
+  public createMicrophoneDownTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration): SisyfosChannelsTimelineObject {
+    return this.buildMicrophoneTimelineObject(blueprintConfiguration, SisyfosFaderState.OFF)
+  }
+
+  public createMicrophoneUpTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration): SisyfosChannelsTimelineObject {
+    return this.buildMicrophoneTimelineObject(blueprintConfiguration, SisyfosFaderState.ON)
+  }
+
+  private buildMicrophoneTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration, isPgm:  SisyfosFaderState): SisyfosChannelsTimelineObject {
```

Comment:
Since `isPgm` is of the type `SisyfosFaderState`, then it would be nice the parameter name reflected that.
So it would be `sisyfosFaderState: SisyfosFaderState`.

Reply:
Has been renamed.